tests/component/test_watchdog.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+
+from nidaqmx import Task
+from nidaqmx.constants import WatchdogAOExpirState, WatchdogCOExpirState
+from nidaqmx.system import Device
+from nidaqmx.system.watchdog import AOExpirationState, COExpirationState
+
+
+@pytest.fixture
+def ai_task(generate_watchdog_task: Task, sim_6363_device: Device) -> Task:
+    """Gets an AI task."""
+    task = generate_watchdog_task()
+    task.ai_channels.add_ai_voltage_chan(
+        f"{sim_6363_device.name}/ai0:3", name_to_assign_to_channel="MyChannel"
+    )
+    return task
```

Comment:
It looks like you didn't end up using this.

Reply:
Removed